waterdata/templates/base.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="{{ config.STATIC_ROOT }}/main.css">
+        <script src="{{ config.STATIC_ROOT }}/bundle.js"></script>
+        <title>WDFN UI Home</title>
+    </head>
+    <body>
+        {% include 'partials/usa_banner.html' %}
+        {% include 'partials/header.html' %}
+        <div class="usa-nav-container">
+            {% block content %}{% endblock %}
+        </div>
+        {% include 'partials/footer.html' %}
+        {% block extra_body %}{% endblock %}
+    </body>
+    {% if config.LIVE_RELOAD_PATH %}
+        <script type="text/javascript">
+            // <![CDATA[  <-- For SVG support
+            if ('WebSocket' in window) {
+                (function() {
+                    function refreshCSS() {
+                        var sheets = [].slice.call(document.getElementsByTagName("link"));
+                        var head = document.getElementsByTagName("head")[0];
+                        for (var i = 0; i < sheets.length; ++i) {
+                            var elem = sheets[i];
+                            head.removeChild(elem);
+                            var rel = elem.rel;
+                            if (elem.href && typeof rel != "string" || rel.length == 0 || rel.toLowerCase() == "stylesheet") {
+                                var url = elem.href.replace(/(&|\?)_cacheOverride=\d+/, '');
+                                elem.href = url + (url.indexOf('?') >= 0 ? '&' : '?') + '_cacheOverride=' + (new Date().valueOf());
+                            }
+                            head.appendChild(elem);
+                        }
+                    }
+                    var socket = new WebSocket('{{ config.LIVE_RELOAD_PATH }}');
+                    socket.onmessage = function(msg) {
+                        if (msg.data == 'reload') window.location.reload();
+                        else if (msg.data == 'refreshcss') refreshCSS();
+                    };
+                    console.log('Live reload enabled.');
+                })();
+            }
+            // ]]>
+        </script>
```

Comment:
This snippet maintains the open connection to the `live-server` instance, that can auto-reload when static assets are recompiled.